oauthenticator/tests/test_gitlab.py

```diff
@@ -37,9 +37,18 @@ def gitlab_client(client):
     )
     return client
 
+def mock_api_version(client, version):
+    def mock_version_response(request):
+        ret = { 'version': version, 'revision': "f79c1794977" }
+        return HTTPResponse(request, 200,
+                            headers={'Content-Type': 'application/json'},
+                            buffer=BytesIO(json.dumps(ret).encode('utf-8')))
+    regex = re.compile(API_ENDPOINT + '/version')
+    client.hosts['gitlab.com'].append((regex, mock_version_response))
 
 async def test_gitlab(gitlab_client):
     authenticator = GitLabOAuthenticator()
+    mock_api_version(gitlab_client, '12.3')
```

Comment:
Mock with a realistic version response, such as `12.2.5-ee`.

This was what my /api/v4/version endpoint returned for example.
```json
{"version":"12.2.5-ee","revision":"e817f2e2"}
```

Reply:
Thanks! Updated